src/app/lib/config/services/zhongwen.js

```diff
@@ -239,6 +239,10 @@ export const service = {
     ...baseServiceConfig,
     lang: `zh-hant`,
     locale: `zh-hant`,
+    mostRead: {
```

Comment:
Are these translations? Should we keep all translation keys in the translations object?

Reply:
yes they are, good point, I'll move all the most-read object into translations object now 👍